### PR TITLE
1

### PR DESCRIPTION
**Pull request by KosSerzh**
https://KosSerzh.github.io/layout_moyo-header/report/html_report/
https://KosSerzh.github.io/layout_moyo-header/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,8 +7,76 @@
...
+      <a href="#Home" class="logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav>
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a 
+              href="#" 
+              class="nav_link is-active"
+              >
```
Comment (VETALEK) [inline src/index.html:36]:
```suggestion
            <a 
              href="#" 
              class="nav_link is-active"
            >
```

need to fix indentation before `>` (everywhere) 😉

File: src/style.css [hunk 2]
```diff
@@ -1 +1,53 @@
...
+  margin: 0;
+  padding: 0 50px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  background-color: #fff;
+}
+
+.logo {
+  padding: 10px;
```
Comment (VETALEK) [inline src/style.css:13]: are you sure you need horizontal padding too?

File: src/style.css [hunk 3]
```diff
@@ -1 +1,53 @@
...
+.nav_list {
+  list-style-type: none;
+  margin: 0;
+  display: flex;
+  flex-direction: row;
+  align-items: baseline;
+}
+
+.nav_item {
+  margin: 0 10px;
```
Comment (VETALEK) [inline src/style.css:25]: your last `.nav_item` doesn't need the margin from the right 😉

**Review by VETALEK — CHANGES_REQUESTED** [review]
Can't see your demo 😭
Have you already performed `npm run deploy`?

image

File: src/style.css (hunk 3 above)
Comment (KosSerzh) [reply src/style.css:25]: Question : I added the class nav_item_last , so why margin-left: 10px doesnt work but margin-right: 0 worked out ?
Thanks !

**Comment by KosSerzh** [comment]
> Can't see your demo 😭 Have you already performed `npm run deploy`?
> 
> image

**Review by VETALEK — CHANGES_REQUESTED** [review]
Still can't see the demo 😭😭😭

File: src/style.css (hunk 3 above)
Comment (VETALEK) [reply src/style.css:25]: didn't get the question 🤔

Comment (VETALEK) [reply src/style.css:25]: i can advise you using pseudoclass `:not(:last-child)`

**Comment by KosSerzh** [comment]
Hey !
1 yes i always run "npm run deploy" after git push , so idk whats wrong
with the demo😭
2 the question was: why on the class="nav_item_last" margin-left: 10 px
didnt work, but margin-righ:0 worked out well ?
Thanks

пт, 10 черв. 2022 р. о 10:12 v.avdyeyenko ***@***.***> пише:

**Comment by KosSerzh** [comment]
Sorry im getting confused - your last comments are regarding my last commit (35 minutes ago) with fixed styles or still regarding the previous one (3 hours ago) ?

**Review by VETALEK — CHANGES_REQUESTED** [review]
you need to reset the `margin` for the `body` 😉

image

File: src/style.css [hunk 4]
```diff
@@ -1 +1,61 @@
 /* Styles go here */
+body {
+  display: block;
+  margin: 8px;
+}
```
Comment (VETALEK) [inline src/style.css:5]:
```suggestion
body {
  margin: 0px;
}
```

like that 😁

**Review by VETALEK — CHANGES_REQUESTED** [review]
Last charge 💪

**Review by VETALEK — APPROVED** [review]
Nice 👌

**Pull request closed without merging**